apps/angular/src/core/services/anime-query-parameters.ts

```diff
@@ -0,0 +1,77 @@
+import { Injectable, inject } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { map, switchMap } from 'rxjs/operators';
+import { Observable, of } from 'rxjs';
+import { Pagination } from '@js-camp/core/models/pagination.model';
+import { Anime } from '@js-camp/core/models/anime';
+import { AnimeQueryParameters } from '@js-camp/core/models/anime-parameters.model';
+import { AnimeQueryParametersMapper } from '@js-camp/core/mappers/anime-parameters.mapper';
+import { AnimeQueryParametersDto } from '@js-camp/core/dtos/anime-parameters.dto';
+
+import { AnimeApiService } from './anime-api.service';
+
+/** Works with anime query parameters. */
+@Injectable({
+	providedIn: 'root',
+})
+export class AnimeQueryParametersService {
+
+	private readonly startPageIndex = 0;
+
+	private readonly defaultPageSize = 5;
```

Comment:
Please use constants outside component.

```suggestion
const START_PAGE_INDEX = 0;
const DEFAULT_PAGE_SIZE = 5;

/** Works with anime query parameters. */
@Injectable({
	providedIn: 'root',
})
export class AnimeQueryParametersService {
```